lute/require/src/bundlevfs.cpp

```diff
@@ -0,0 +1,127 @@
+#include "lute/bundlevfs.h"
+
+#include "Luau/Common.h"
+
+#include <string>
+
+static const std::string BUNDLE_PREFIX = "@bundle/";
```

Comment:
```suggestion
#include "Luau/Common.h"
#include "Luau/DenseHash.h"

#include <string>
#include <string_view>

constexpr std::string_view kBundlePrefix = "@bundle/";
```
to match existing constant style in other places in the code